server/build/gzip.js

```diff
@@ -8,7 +8,8 @@ export default async function gzipAssets (dir) {
 
   const coreAssets = [
     path.join(nextDir, 'commons.js'),
-    path.join(nextDir, 'main.js')
+    path.join(nextDir, 'main.js'),
+    path.join(nextDir, 'static-bundle.js')
```

Comment:
@arunoda since the execution of the files is synchronous, is it really better to have 3 files?